hack/cluster-clean.sh

```diff
@@ -83,64 +79,20 @@ function delete_resources() {
     # Namespaced resources
     for i in ${namespaces[@]}; do
         for label in ${labels[@]}; do
-            _kubectl -n ${i} delete deployment -l ${label}
-            _kubectl -n ${i} delete ds -l ${label}
-            _kubectl -n ${i} delete rs -l ${label}
-            _kubectl -n ${i} delete pods -l ${label}
-            _kubectl -n ${i} delete services -l ${label}
-            _kubectl -n ${i} delete pvc -l ${label}
-            _kubectl -n ${i} delete rolebinding -l ${label}
-            _kubectl -n ${i} delete roles -l ${label}
-            _kubectl -n ${i} delete serviceaccounts -l ${label}
-            _kubectl -n ${i} delete configmaps -l ${label}
-            _kubectl -n ${i} delete secrets -l ${label}
-            _kubectl -n ${i} delete jobs -l ${label}
+            _kubectl -n ${i} delete deployment,ds,rs,pods,services,pvc,rolebinding,role,serviceaccounts,configmaps,secrets,jobs -l ${label}
         done
     done
 
     # Not namespaced resources
     for label in ${labels[@]}; do
-        _kubectl delete validatingwebhookconfiguration -l ${label}
-        _kubectl delete pv -l ${label}
-        _kubectl delete clusterrolebinding -l ${label}
-        _kubectl delete clusterroles -l ${label}
-        _kubectl delete customresourcedefinitions -l ${label}
-
-        # W/A for https://github.com/kubernetes/kubernetes/issues/65818
-        _kubectl delete apiservices -l ${label} --wait=false
-
-        _kubectl get apiservices -l ${label} -o=custom-columns=NAME:.metadata.name,FINALIZERS:.metadata.finalizers --no-headers | grep foregroundDeletion | while read p; do
-            local arr=($p)
-            local name="${arr[0]}"
-            _kubectl -n ${i} patch apiservices $name --type=json -p '[{ "op": "remove", "path": "/metadata/finalizers" }]'
-        done
+        _kubectl delete apiservices,clusterroles,clusterrolebinding,customresourcedefinitions,pv,validatingwebhookconfiguration -l ${label}
```

Comment:
I think it is important to keep the deletion order especially removing CRDs before APIServices, at least it was important in the past.

Personally I try to follow Helm order: https://github.com/helm/helm/blob/release-3.7/pkg/releaseutil/kind_sorter.go#L31, deletion is done in the reverse order of installation.

Reply:
After looking at Helm code, I just noticed my comment is stupid anyway, APIServices are removed first in Helm.

Reply:
I did not follow: do you suggest to change anything? @dollierp ?

Reply:
I was suggesting you to follow the deletion order of Helm but you were already following it. So no :)

Reply:
Would you feel comfortable grating your lgtm?